app/styles/components/_list-item.scss

```diff
@@ -404,6 +404,47 @@ $sd-ListItem-column-border: scale-color($grayNeutral, $alpha: -82%);
     }
 }
 
+
+.avatar {
```

Comment:
You don't need to change this to `.sd-avatar`?

Reply:
Nope. `sd-avatar` classnames are only for a react component. I'm keeping this for compatibility where react component isn't used.

Reply:
👍